### PR TITLE
Feature/df 860

### PR DESCRIPTION
**Pull request by fabiangotzen**
Currently running (together with adjusted configs) on http://staging.job-desk.ch/#/

### CODE REVIEW

File: infrastructure/src/main/java/ch/admin/seco/jobs/services/jobdesk/infrastructure/elasticsearch/ElasticsearchService.java [hunk 1]
```diff
@@ -0,0 +1,4 @@
+package ch.admin.seco.jobs.services.jobdesk.infrastructure.elasticsearch;
+
+public class ElasticsearchService {
```
Comment (birom) [inline infrastructure/src/main/java/ch/admin/seco/jobs/services/jobdesk/infrastructure/elasticsearch/ElasticsearchService.java]: I think we do not need this class.

File: infrastructure/src/main/java/ch/admin/seco/jobs/services/jobdesk/infrastructure/elasticsearch/ElasticsearchIndexService.java [hunk 2]
```diff
@@ -36,7 +36,8 @@
 @Service
 public class ElasticsearchIndexService {
 
-    public static final String INDEX_NAME = "jobdesk";
+    public static final String JOB_INDEX_NAME = "job";
+    public static final String LOCATION_INDEX_NAME = "location";
     public static final String JOBDESK_JOB_TYPE = "job";
```
Comment (birom) [inline infrastructure/src/main/java/ch/admin/seco/jobs/services/jobdesk/infrastructure/elasticsearch/ElasticsearchIndexService.java]: I suggest to rename the type to ``doc``. We also use that in the job-ad-service. In the latest ES version, we can have only one type in a per index.

File: infrastructure/src/main/java/ch/admin/seco/jobs/services/jobdesk/infrastructure/elasticsearch/ElasticsearchService.java (hunk 1 above)
Comment (fabiangotzen) [reply infrastructure/src/main/java/ch/admin/seco/jobs/services/jobdesk/infrastructure/elasticsearch/ElasticsearchService.java]: oh, pushed it by accident. removed it now.

File: infrastructure/src/main/java/ch/admin/seco/jobs/services/jobdesk/infrastructure/elasticsearch/ElasticsearchIndexService.java (hunk 2 above)
Comment (fabiangotzen) [reply infrastructure/src/main/java/ch/admin/seco/jobs/services/jobdesk/infrastructure/elasticsearch/ElasticsearchIndexService.java]: Adjusted the type to doc for location & job index.

**Pull request merged**